.github/workflows/sync-from-fork.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           git remote add upstream ${UPSTREAM_URL}
           git fetch upstream
 
-          FILES_TO_CHECK=("assets/assets.json"  "assets/resources/scriptlets.js"  "src/web_accessible_resources"  "src/js/redirect-resources.js")
+          FILES_TO_CHECK=("assets/assets.json"  "assets/resources/scriptlets.js"  "assets/resources/run-at.js" "assets/resources/safe-self.js" "src/web_accessible_resources"  "src/js/redirect-resources.js")
```

Comment:
we probably want to use the whole `assets/resources` directory similar to what we do for `src/web_accessible_resources`